introduction.md

```diff
@@ -0,0 +1,2 @@
+Manjunatha Kulkarni learning git basics. My git account id is OpenMan18
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I am new to code school. mastering GitHub is my first course here. Mastering GitHub is my current favorite.

Reply:
Added my favorite course.